PosteAPI/views.py

```diff
@@ -286,10 +288,13 @@ def get(self, request, pk):
             folders = shared_folders | owned_folders
 
         serializer = FolderSerializer(folders, many=True)
-        return Response({"success": True,"folders": serializer.data}, status=status.HTTP_200_OK)
+        return Response({"success": True, "folders": serializer.data}, status=status.HTTP_200_OK)
 
 
 class PostAPI(APIView):
+    authentication_classes = [TokenAuthentication]
```

Comment:
Great. This requires a token be passed and verifies the user is authenticated to perform the actions they're attempting

Reply:
I was actually stumped on this for a while. And then I had to merge your deletePost in the backend to use as a reference!